reactapp/config-overrides.js

```diff
@@ -1,69 +1,70 @@
-// Import the Hyva Inheritance plugin
-const HyvaInheritancePlugin = require('./webpack-hyva-inheritance-plugin');
 const path = require('path');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { aliasDangerous } = require('react-app-rewire-alias/lib/aliasDangerous');
+const HyvaInheritancePlugin = require('./webpack-hyva-inheritance-plugin');
 
-// Define the original Hyva sources as a parent
-// @todo: Make this configurable through `.env`
-const parentPath = path.resolve("../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/reactapp/src");
-
-module.exports = {
-    webpack: function(config, env) {
-        const isEnvDevelopment = env === 'development';
-        const isEnvProduction = env === 'production';
-        const filename = isEnvProduction
-            ? '../../view/frontend/web/js/react-checkout.js'
-            : isEnvDevelopment && 'static/js/bundle.js';
-        const chunkFilename = isEnvProduction
-            ? '../../view/frontend/web/js/[name].chunk.js'
-            : isEnvDevelopment && 'static/js/[name].chunk.js';
+const hyvaCheckoutVendorPath =
+  '../../../../../vendor/hyva-themes/magento2-hyva-checkout/src/reactapp/src';
```

Comment:
`path.resolve` is used to create an absolute path instead of a relative one

Not sure if this makes sense here.

Reply:
If you check line 54 in that file, you will see I am using the absolute path in the plugin. But, it is true that i used relative path in `aliasMap`. I will try to use the absolute path there and check whether it works correctly.

Reply:
Fixed this issue. Now we are using absolute path everywhere.